modules/snooker/src/test/ts/browser/TableAdjustmentsTest.ts

```diff
@@ -47,19 +70,46 @@ UnitTest.test('TableAdjustmentsTest', () => {
   </tbody>
   </table>`);
 
+  const pixelTableWithColGroup = () => SugarElement.fromHtml<HTMLTableElement>(`<table style="border-collapse: collapse; width: 400px;" border="1">
+  <colgroup>
+  <col style="width: 96.75px;">
```

Comment:
These values don't look right given our testing the other day, as the col width uses the outer width (eg with borders and padding). I suspect these should be `100px` instead?